lib/logstash/plugin_mixins/http_client.rb

```diff
@@ -109,8 +109,7 @@ def client_config
       pool_max: @pool_max,
       pool_max_per_route: @pool_max_per_route,
       cookies: @cookies,
-      keepalive: @keepalive,
-      verify: @ssl_certificate_validation
```

Comment:
I had to move it to a lower spot since it needs to be nested under the `:ssl` key.